noq_django/backend/forms.py

```diff
@@ -76,7 +76,21 @@ class Meta:
 class UserForm(forms.ModelForm):
```

Comment:
Nånting är fel. Vet inte vad eller var. Om jag skapar användare (user), skapas ingen client. Om jag tittar på Användarelistan visas nya användare, men Client listan har inte användaren. Man kan logga in som användare, men man kan inte boka plats via api/user/request_booking.

Reply:
Ok. Kanske det är som det ska i så fall. Men handläggare måste skapa både user och client innan brukare kan boka sovplats. Ska testa mer på fredag. Men ok från review perpektiv baserad på din kommentar.